build/Settings.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
```

Comment:
@filipw @JoeRobich anyone have an issue with this change?

Reply:
I don't have a problem with it.

Reply:
good because the latest library has lots of records... lots of with statements 🤣

Reply:
one small issue with this is that some features of C# 9 e.g. positional records or `init` are not available without .NET 5.0, so it is easy to fall into the trap of using something that won't compile 😄

Reply:
records have been working fine for me, the only gotcha is much like NRT you have to provide an interop type.  
https://github.com/OmniSharp/csharp-language-server-protocol/blob/81a38cca80a4e35ec198de75dbab5a5571aae82a/src/JsonRpc/Nullable/Records.cs

Reply:
I've tried to use `or` patterns several times now, please give them to me :)